### PR TITLE
jiaojinlong

### PR DESCRIPTION
**Pull request by DragonJiao**

### CODE REVIEW

**Comment by YiqinZhao** [comment]
@DragonJiao Thanks for your PR, however, I cannot merge it at this time due to serval problems:

1. Please include a Chinese version biography.
1. Please remove characters like `、` in english paragraph, this mark is only used in Chinese.
1. Please add more information for your project **[Project] Hierachical Attention for Speech Emotion Recognition**

File: content/en/author/jiaojinlong/_index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+#   icon = "cv"
+
+#   icon_pack = "ai"
+
+#   link = "files/cv.pdf"
+
++++
+
+I am a senior undergraduate software engineering student at Tianjin Normal University, Tianjin, China. I am a key member of the Cognitive and Affective Computing Lab at my TJNU, advised by Prof. Ziping Zhao.More info about me: https://dragonjiao.github.io/
```
Comment (YiqinZhao) [inline content/en/author/jiaojinlong/_index.md]: A `senior` student usually means a forth-year student at collage. I suppose you need another word at here.

File: content/zh/author/jiaojinlong/_index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+# [[social]]
+
+#   icon = "cv"
+
+#   icon_pack = "ai"
+
+#   link = "files/cv.pdf"
+
++++
+I am a senior undergraduate software engineering student at Tianjin Normal University, Tianjin, China. I am a key member of the Cognitive and Affective Computing Lab at my TJNU, advised by Prof. Ziping Zhao.More info about me: https://dragonjiao.github.io/
```
Comment (YiqinZhao) [inline content/zh/author/jiaojinlong/_index.md]: How about add some Chinese biography

**Comment by YiqinZhao** [comment]
@DragonJiao I believe we still need some work to reach a merge.

File: content/zh/author/jiaojinlong/_index.md (hunk 2 above)
Comment (DragonJiao) [reply content/zh/author/jiaojinlong/_index.md]: I can not understand it well

Comment (YiqinZhao) [reply content/zh/author/jiaojinlong/_index.md]: 加上中文版的自我介绍

**Comment by YiqinZhao** [comment]
@DragonJiao Thanks. Another minor issue, **iOS Club** should be written in this way. This is not a big deal, I have merged and I will fix it latter.

**Pull request merged**